apps/kube-prometheus-stack/values.yaml

```diff
@@ -0,0 +1,6 @@
+resources:
+  limits:
+    memory: 200Mi
+  requests:
+    cpu: 100m
+    memory: 100Mi
```

Comment:
please go over the other values in the dependant charts and add relevant values, like:

* storage configuration
* retention
* monitored components (depending on what you see in the cluster kube-system namespace)

Reply:
Pushed an update.
WDYT?

Reply:
I think everything needs to be under `kube-prometheus-stack:` - did you check that the configurations are actually applied when you run it locally?

Reply:
You're totally right.
Fixed.